kotlin-remix-run-router/src/main/generated/remix/run/router/createStaticHandler.kt

```diff
@@ -0,0 +1,11 @@
+@file:JsModule("@remix-run/router")
+@file:JsNonModule
+
+@file:Suppress(
+    "NON_EXTERNAL_DECLARATION_IN_INAPPROPRIATE_FILE",
+    "NAME_CONTAINS_ILLEGAL_CHARS",
+)
+
+package remix.run.router
+
+external fun createStaticHandler(routes: Array<AgnosticRouteObject>, opts: CreateStaticHandlerOpts = definedExternally): StaticHandler
```

Comment:
`ReadonlyArray`?

Reply:
There is no signs in source code that this array should be readonly, I will perform revision and add list of places where it would be great to make arrays readonly.